app/views/schemes/check_answers.html.erb

```diff
@@ -9,7 +9,7 @@
       <h2 class="govuk-visually-hidden">Scheme</h2>
       <dl class="govuk-summary-list">
         <% @scheme.check_details_attributes.each do |attr| %>
-          <%= render partial: "scheme_summary_list_row", locals: { scheme: @scheme, attribute: attr, change_link: @scheme.confirmed? ? scheme_edit_name_path(@scheme) : scheme_details_path(@scheme, check_answers: true) } %>
+          <%= render partial: "scheme_summary_list_row", locals: { scheme: @scheme, attribute: attr, change_link: current_user.support? ? scheme_details_path(@scheme, check_answers: true) : scheme_edit_name_path(@scheme) } unless attr[:name] == "Status" %>
```

Comment:
won't this redirect to the edit name path for an unconfirmed (incomplete) scheme viewed by a data coordinator? I think we should still reirect to the details path in that case

Reply:
Good spot!